presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -368,6 +366,10 @@ private static boolean shouldUseFileSplitsFromInputFormat(InputFormat<?, ?> inpu
         return splitList;
     }
 
+    private String getCorruptBucketingMsg() {
```

Comment:
This will fail Checkstyle. Please see Travis failures.

Reply:
Don't abbreviate. This should be `message` not `msg`

Reply:
Removed the method as per Raghav.